src/Controller/Component/CrudComponent.php

```diff
@@ -448,7 +448,7 @@ public function isActionMapped(?string $action = null): bool
     public function on(array|string $events, callable $callback, array $options = []): void
     {
         foreach ((array)$events as $event) {
-            if (!strpos($event, '.')) {
+            if (!str_contains($event, '.')) {
```

Comment:
Similar here, is this more performant? Otherwise seems a bit pointless

Reply:
Same as the other comment

Reply:
If that is a good and clear explanation, can you please close the conversation?